Annotation/alternative_golden_set.json

```diff
@@ -147,7 +147,7 @@
                     }
                 ],
                 "type": "alternative",
-                "clause": "",
+                "clause": "whether Rocco must return to New York, and his mother, or whether he can stay in London with his father",
                 "embedded clauses": []
             }
         ]

```

Comment:
I think there's an argument here that it could be two polar EC or a single alternative EC, it's ambiguous

Reply:
I agree that it's a little ambiguous but it seems more presented as an alternative imo - if he can't stay with his father in London, he will have to return to his mother in NY (2 possible situations)